### PR TITLE
Mat 5750/link string component

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-5750](https://jira.cms.gov/browse/MAT-5750)
(Optional) Related Tickets:

### Summary
linking string component
### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/common/string/StringField.tsx [hunk 1]
```diff
@@ -12,41 +12,42 @@ export interface StringFieldProps {
   label: string;
   fieldValue: string;
   canEdit: boolean;
+  onStringValueChange: Function;
   required?: boolean;
 }
 
 const StringField = ({
```
Comment (adongare) [inline src/components/common/string/StringField.tsx]: Can we change the name of the component to StringInput just to be consistent with all other primitive components?

File: src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/DisplayMultipleAttributeInputs.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
-import React from "react";
+import React, { useState } from "react";
 import IdentifierInput from "../../../../../../../common/Identifier/IdentifierInput";
 import { Identifier } from "cqm-models";
+import StringField from "../../../../../../../common/string/StringField";
 
 const DisplayMultipleAttributeInputs = ({
```
Comment (adongare) [inline src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/DisplayMultipleAttributeInputs.tsx]: this component seems more specific to Entities. Either we make it dynamic to display multiple attributes based on their types or change the name to Entity

File: src/components/common/string/StringField.tsx (hunk 1 above)
Comment (sb-prateekkeerthi) [reply src/components/common/string/StringField.tsx]: thanks for the comment, I'll change it

File: src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/DisplayMultipleAttributeInputs.tsx (hunk 2 above)
Comment (sb-prateekkeerthi) [reply src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/DisplayMultipleAttributeInputs.tsx]: thanks for the comment, I'll change it

**Review by adongare — APPROVED** [review]

**Pull request merged**